resources/function_help/json/geometry_overlay_contains

```diff
@@ -2,15 +2,25 @@
   "name": "geometry_overlay_contains",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description": "Performs a spatial join of type CONTAINS. This returns an array of results of an expression evaluated on features from a different layer that CONTAINS the current feature, or, if no expression if provided, simply returns whether at least one feature from the other layer CONTAINS the current feature.",
-  "arguments": [
+  "description": "Performs a spatial join of type CONTAINS. This returns whether at least one feature from a layer contains the current feature, or an array of results of an expression evaluated using features that contain the current feature.",
```

Comment:
Do we have a link we can point to, explaining what a contains spatial join is? something like https://postgis.net/docs/ST_Contains.html? The GEOS website is hard to browse to me.

Reply:
Do we have something in our docs?

Reply:
The "only" place in docs we expose this kind of predicates is at [extract by location alg](https://docs.qgis.org/testing/en/docs/user_manual/processing_algs/qgis/vectorselection.html#extract-by-location) and alike (reminds me that we should adapt this to the "join by location"-based algs)

The "source" is available as html at https://docs.qgis.org/testing/en/docs/user_manual/processing_algs/qgis/qgis_algs_include.html but this is a hidden,  internal and "garbage" file I'm not sure we want to expose.

Reply:
Too bad. I think it would be good to cover such concepts somewhere in our documentation but am not experienced enough with the structure to advice where.

Reply:
To follow up on this, ideally (IMHO) we would add this to our docs but linking to postgis sounds fine too, it's correct in content but weird in context.

Reply:
Quickly "browsing" the docs, I can't see where we would fluently add this but I'm surprised no project already sketched these concepts.
Yep, I can see the weirdness you mention. How about:
```suggestion
  "description": "Performs a spatial join of type CONTAINS. This returns whether at least one feature from a layer contains the current feature, or an array of results of an expression evaluated using features that contain the current feature.<br>This function follows the PostGIS <a href='https://postgis.net/docs/ST_Contains.html'>ST_CONTAINS</a> function.",
```

Reply:
"...the GEOS functionality which is documented in PostGIS"? Would be more accurate but maybe too complex to read?